### PR TITLE
Fix name + Details

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
Additional requests:
- Please activate the issue tab in this repository.
- Please correct the URL in the description. It still says MMM-Pages and not MMM-pages.

### CODE REVIEW

**Comment by sdetweil** [comment]
done and done

**Comment by KristjanESPERANTO** [comment]
Thanks! :smiley:

Here is still a "P":

Screenshot from 2024-06-24 22-42-24

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ have aptly described your issue in words, feel free to ignore/delete this sectio
 ```json
 (Paste the part of the config file here)
 ```
- - Please post any errors you see about MMM-Pages in the console (Hit F12 > Console when the Magic Mirror window is focused), or write **None** if there aren't any:
+ - Please post any errors you see about MMM-pages in the console (Hit F12 > Console when the Magic Mirror window is focused), or write **None** if there aren't any:
```
Comment (KristjanESPERANTO) [inline .github/ISSUE_TEMPLATE/Bug_report.md:32]: You've forgot this part.

**Comment by KristjanESPERANTO** [comment]
@sdetweil Did you noticed my last two comments? 🙂

**Comment by sdetweil** [comment]
sorry, i thought u did that work in the prior pr.

**Pull request closed without merging**